spec/features/capybara_homepage_spec.rb

```diff
@@ -16,15 +16,18 @@
 describe 'homepage' do
   after { Warden.test_reset! }
   it 'validates presence of header and footer on homepage' do
+    # The theme at U of A doesn't match the upstream theme
+
     visit 'http://0.0.0.0:3000'
-    page.should have_content('Sample Content')
+    # Not relevant: page.should have_content('Sample Content')
```

Comment:
If not needed, I would remove instead of commenting out. But guess this for for upstream or something?